Makefile

```diff
@@ -13,15 +13,16 @@ clean-pyc:
 	find . -name '*~' -exec rm -f {} +
 
 build: clean
-	python setup.py sdist bdist_wheel
+	python -m build
```

Comment:
@ian-ross what does this do?

Reply:
It uses [this thing](https://pypa-build.readthedocs.io/en/latest/) to wrap the build process. More recent versions of pip complain about packages with only a `setup.py`, so I needed to add a `pyproject.toml`, and this `build` package is a nice front-end to the whole `pyproject.toml`-drive package build approach. (It works with other build systems like Poetry as well, so you can just say `python -m build` to build your distribution whatever build system you're using.)

Reply:
oh cool! do I need to install poetry? 

I was curious about this because when i tried doing `make build` I get a `No module named build`

Reply:
Ha. You're totally right. That's a mistake.

You don't need to install Poetry. The build still uses `setuptools` (you can see that in the [`pyproject.yml` file](https://github.com/dotenv-org/python-dotenv-vault/blob/master/pyproject.toml#L2-L3)). However, you *do* need to have the [build package](https://pypi.org/project/build/) installed. I have this installed because I use Poetry on some other projects and Poetry depends on `build` so I get it for free. It's not working for you because the `build` package isn't installed by default, and I forgot to add something to the Makefile to install it. There's a [line in the CI setup](https://github.com/dotenv-org/python-dotenv-vault/blob/master/.github/workflows/python-publish.yml#L32) that installs it, but I need to add something to the Makefile as well.

I'll do that right now and make a new release. Thanks for looking at this!

Reply:
Fixed in v0.5.2! Thanks again!